src/Main.cpp

```diff
@@ -57,8 +58,10 @@ static const struct OptionTemplate
     {'l', "don't use previous results.", OptionType::kBool, offsetof(DriverOptions, m_DontReusePreviousResults), "Builds the requested target from scratch"},
     {'w', "spammy-verbose", OptionType::kBool, offsetof(DriverOptions, m_SpammyVerbose), "Enable spammy verbose build messages"},
     {'D', "debug", OptionType::kBool, offsetof(DriverOptions, m_DebugMessages), "Enable debug messages"},
+    {'k', "continue-on-failure", OptionType::kBool, offsetof(DriverOptions, m_ContinueOnFailure), "Build as much as possible after the first error"},
```

Comment:
I don't have a strong opinion on this. I like picking up queues from other systems but at the same time `--keep-going` is maybe not the best name I've seen. However, it would match that short `k` ;)

@lucasmeijer what do you think? If you don't care I will go with what @mostynb likes.

Reply:
I think continue-on-failure does a good job describing what it does,  whereas keep-going does not